src/views/Contacts.vue

```diff
@@ -315,6 +325,56 @@ export default {
 			}
 		},
 
+		/**
+		 * Download vcard promise as vcard file
+		 *
+		 * @param {Object} vcardPromise object to be downloaded
+		 */
+		downloadVcardPromise(vcardPromise) {
+			vcardPromise.then(response => {
+				const blob = new Blob([response.data], { type: 'text/vcard' })
+				const url = URL.createObjectURL(blob)
+				const link = document.createElement('a')
+				const filename = moment().format('YYYY-MM-DD_HH-mm') + '_' + response.groupName + '.vcf'
+				link.href = url
+				link.download = filename
+				link.click()
+			})
+		},
+
+		/**
+		 * Download group of contacts
+		 *
+		 * @param {Object} group of contacts to be downloaded
+		 */
+		downloadGroup(group) {
+			// get grouped contacts
+			let groupedContacts = {}
+			group.contacts.map((key) => {
+				const id = this.contacts[key].addressbook.id
+				groupedContacts = Object.assign({
+					[id]: {
+						addressbook: this.contacts[key].addressbook,
+						contacts: []
+					}
+				}, groupedContacts)
+				groupedContacts[id].contacts.push(this.contacts[key].url)
+			})
+			// create vcard promise with the requested contacts
+			const vcardPromise = Promise.all(
+				Object.keys(groupedContacts).map(key => {
+					return groupedContacts[key].addressbook.dav.addressbookMultigetExport(groupedContacts[key].contacts)
+				}))
+				.then(response => {
+					return {
```

Comment:
Please use the concise body syntax here as well. Please note that you will have to put the object into parentheses in this case.